### PR TITLE
Update privacy notice

### PR DESCRIPTION
**Pull request by wyattr-spec**
Updated privacy notice

### What
Privacy notice updated including new sections on user research.

### Jira

https://technologyprogramme.atlassian.net/jira/software/projects/DDJTS/boards/251?assignee=628b458a40b157006f721ef5&selectedIssue=GW-1155

### CODE REVIEW

File: source/privacy-notice.html.erb [hunk 1]
```diff
@@ -10,166 +10,145 @@ description: Find details about the GovWifi privacy policy and how we collect an
...
-          <br/>
-          70 Whitehall
-          <br/>
-          London
-          <br/>
-          SW1A 2AS
-        </p>
+<p class="govuk-body">We require this information to ensure that our research is varied and representative of our user groups.</p> 
+
+<p class="govuk-body">This data gived us insights from end users and organisations to help us identify ways we can improve GovWifi. It allows us to capture pain points and to broaden our knowledge of how the service is used and seen.</p>
```
Comment (szd55gds) [inline source/privacy-notice.html.erb:70]: gives

File: source/privacy-notice.html.erb [hunk 2]
```diff
@@ -10,166 +10,145 @@ description: Find details about the GovWifi privacy policy and how we collect an
...
+Processing is necessary for the performance of a task carried out in the public interest or in the exercise of official authority vested in the data controller. In this case that is the running of wifi services.
+          </p>
+  <h2 class="govuk-heading-m" id="user-research">For user research</h2>
+     <p class="govuk-body">The initial use of personal data to send surveys (when data subjects sign up for GovWifi credentials) is on a lawful basis of legitimate interest.</p>
+
+<p class="govuk-body">All other processing for user research is on a lawful basis of consent.</p>
+
+   <h2 class="govuk-heading-m" id="recipients">Recipients</h2>
+ 
+<p class="govuk-body">Your personal data may be shared with other Government department in specific circumstances such as investigations: </p>
```
Comment (szd55gds) [inline source/privacy-notice.html.erb:89]: departments

File: source/privacy-notice.html.erb [hunk 3]
```diff
@@ -10,166 +10,145 @@ description: Find details about the GovWifi privacy policy and how we collect an
...
+          <li>Legal investigations</li>
+  <li>Cyber Security investigations</li>
+  <li>HR employment investigations</li>
+  <li>If required by court order</li></ul>
+
+<p class="govuk-body">We will not:</p>
+<ul class="govuk-list govuk-list--bullet">
+          <li>sell or rent your data to third parties</li>
+<li>share your data with third parties for marketing purposes</li>
+<li>We do not share data that monitor or track employees</li></ul>
```
Comment (szd55gds) [inline source/privacy-notice.html.erb:100]: Don't need the "We do not"
change monitor into monitors
change track into tracks

Comment (wyattr-spec) [reply source/privacy-notice.html.erb]: can you re-approve the PR, ta.

**Pull request merged**